assets/styles/index.scss

```diff
@@ -22,3 +22,19 @@
     color: white;
   }
 }
+
+.docs {
+  display: flex;
+  grid-gap: 2em;
```

Comment:
the grid gap on the old developer page is actually much bigger. Not sure if that's preferred? Double space seemed enough to me